slides/01_intro.qmd

```diff
@@ -5,6 +5,17 @@ description: "Introduction and outline of program.</br>[Photo CC0 by Mabel Amber
 image: thumbnails/hello.jpg
 ---
 
+<!--
+Creators (initial authors): 
+Daniel Sabanes, https://github.com/danielinteractive/ and www.linkedin.com/in/danielsabanesbove/
+Friedrich Pahlke, https://github.com/fpahlke/ and www.linkedin.com/in/pahlke/
+Kevin Kunzmann, https://github.com/kkmann/ and https://www.linkedin.com/in/kevin-kunzmann-6486a11bb/
+License: This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License.
+To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/.
+The source files are hosted at https://github.com/kkmann/workshop-r-swe/.
+Important: To use this work you must provide the name of the creators (initial authors), a link to the material, a link to the license, and indicate if changes were made.
+--> 
```

Comment:
This seems duplicate with the License information at the end of the file?
For maintenance, it would be better to avoid duplication. If it is important to have this at the top, then we can just move the License slide to the top.

Reply:
I agree that we should avoid redundancy. The idea was that if someone removes the license slide, we still have the information in the source. Assuming no one removes the slide, it's okay to delete the information in the source.

Reply:
Yeah nobody may remove this slide. Just want to minimize maintenance burden later. Thanks Friedrich